Products/Zuul/infos/template.py

```diff
@@ -488,6 +488,9 @@ def getRateOptions(self):
             if self._object.isCounter():
                 rateOptions['counter'] = True
                 rateOptions["resetThreshold"] = 1
+            if self._object.rrdtype == "DERIVE" and str(self.rrdmin) != '0':
```

Comment:
str(self.rrdmin) take here because of this comparison:
https://github.com/zenoss/zenoss-prodbin/blob/develop/Products/ZenModel/RRDDataPoint.py#L200-200